database.py

```diff
@@ -35,6 +35,12 @@ class ReportGithubIssues(Base):
     __table__ = Table('report_github_issues', Base.metadata, autoload=True)
 
 
+'''
+class ReportJiraQARequests(Base):
```

Comment:
Commenting for now because the table is not in the staging or production database yet